$OpenBSD: patch-configure,v 1.24 2015/06/23 15:14:52 feinerer Exp $

Unbreak build on powerpc (relocation truncated to fit: R_PPC_GOT16...)

--- configure.orig	Thu Apr  9 01:16:36 2015
+++ configure	Sat May  2 11:47:48 2015
@@ -26276,7 +26276,7 @@ if test "${GCC}" = yes; then
 ## has 32k and so can use -fpic.
 ## However, although the gcc docs do not mention it, it seems s390/s390x
 ## also supports and needs -fPIC
-    sparc*|ppc64|powerpc64|s390*)
+    sparc*|ppc64|powerpc*|s390*)
       cpicflags="-fPIC"
       ;;
     *)
@@ -26287,7 +26287,7 @@ if test "${GCC}" = yes; then
 fi
 if test "${G77}" = yes; then
   case "${host_cpu}" in
-    sparc*|ppc64|powerpc64|s390*)
+    sparc*|ppc64|powerpc*|s390*)
       fpicflags="-fPIC"
       ;;
     *)
@@ -26297,7 +26297,7 @@ if test "${G77}" = yes; then
 fi
 if test "${GXX}" = yes; then
   case "${host_cpu}" in
-    sparc*|ppc64|powerpc64|s390*)
+    sparc*|ppc64|powerpc*|s390*)
       cxxpicflags="-fPIC"
       ;;
     *)
@@ -38862,7 +38862,7 @@ fi
 ## use this to pick out gfortran (even though it is unreliable).
 if test "${ac_cv_fc_compiler_gnu}" = yes; then
   case "${host_cpu}" in
-    sparc*|ppc64|powerpc64|s390*)
+    sparc*|ppc64|powerpc*|s390*)
       fcpicflags="-fPIC"
       ;;
     *)
